src/scripts/backupScript.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# 
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+# Input the pathname for the desired backup directory
```

Comment:
It might be good to put in a comment at the start that the OED database Docker container must be running for this to work. It also assumes it is the only database container running (seems likely).

Reply:
I've included a message at the top that mentions this now, thank you for that suggestion.